### PR TITLE
 2.5.0 release

### PR DESCRIPTION
**Pull request by sjsrey**
Any new package highlights can be added to `CHANGELOG.md`.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,269 @@
...
+
+Many thanks to all of the following individuals who contributed to this release:
+
+
+ - Arfon Smith
+ - Dani Arribas-Bel
+ - Eli Knaap
+ - Elliott Sales De Andrade
+ - James Gaboardi
+ - Jkoschinsky
```
Comment (jGaboardi) [inline CHANGELOG.md:256]: Should we have Julia's GitHub handle or her name? @Jkoschinsky
* same for:
  * Mragank @Mgeeeek
  * Xin (Selena) vs. Xin @xf37

And we should add @ljwolf.

File: requirements_docs.txt [hunk 2]
```diff
@@ -1,7 +1,7 @@
 sphinx>=1.4.3
 sphinxcontrib-napoleon
 sphinx-gallery
-sphinxcontrib-bibtex
+sphinxcontrib-bibtex<2.0.0
```
Comment (jGaboardi) [inline requirements_docs.txt:4]: I would say we can remove this pin (and the `>=1.4.3` from `sphinx`), but that would require adding [this](https://github.com/pysal/spaghetti/blob/888047b1efb93a09e30653589cee383ef54d2958/docs/conf.py#L37) to `docs/conf.py` (which seems like doesn't exist?).

**Comment by jkoschinsky** [comment]
either way works re. my name.
Thx!
________________________________

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,269 @@
...
+* [#141:](https://github.com/pysal/tobler/pull/141) add pycnophylactic interpolation 
+* [#95:](https://github.com/pysal/tobler/issues/95) Add Pycnophylactic Interpolation 
+* [#79:](https://github.com/pysal/tobler/issues/79) sentinel data 
+* [#135:](https://github.com/pysal/tobler/pull/135) ENH: support categorical variables in area_interpolate 
+* [#96:](https://github.com/pysal/tobler/issues/96) vector data as a dasymetric mask 
+* [#132:](https://github.com/pysal/tobler/pull/132) bump version 
+
+
+<a name="mapclassify"></a>
+### mapclassify
```
Comment (martinfleis) [inline CHANGELOG.md:189]: @sjsrey do you want to cut a release of mapclassify with https://github.com/pysal/mapclassify/pull/110 and https://github.com/pysal/mapclassify/pull/114?

File: requirements_docs.txt (hunk 2 above)
Comment (martinfleis) [reply requirements_docs.txt:4]: https://github.com/pysal/pysal/blob/master/docsrc/conf.py ?

**Comment by ljwolf** [comment]
I'd like to get esda released as well to make it in here. I'm waiting on a merge at [esda/#133](https://github.com/pysal/esda/pull/133), and I'll then cut the release myself.

File: CHANGELOG.md (hunk 1 above)
Comment (MgeeeeK) [reply CHANGELOG.md:256]: I'll prolly change my weird username, sorry for the trouble :'(

Comment (sjsrey) [reply CHANGELOG.md:256]: > Should we have Julia's GitHub handle or her name? @jkoschinsky
> 
>     * same for:
>       
>       * Mragank @MgeeeeK
>       * Xin (Selena) vs. Xin @xf37
> 
> 
> And we should add @ljwolf.

The contributor list is autogenerated in https://github.com/pysal/pysal/blob/master/tools/110-gitcount-tables.ipynb

There is a mapping to link an id to a preferred name. PRs to adjust would be welcomed.

File: CHANGELOG.md (hunk 3 above)
Comment (sjsrey) [reply CHANGELOG.md:189]: will do so tonight.

File: requirements_docs.txt (hunk 2 above)
Comment (jGaboardi) [reply requirements_docs.txt:4]: > https://github.com/pysal/pysal/blob/master/docsrc/conf.py ?

Yeah, I guess I missed that. I'm not sure how the actual PySAL docs get built, or if they are simply built at the submodule-level.

**Review by darribas — APPROVED** [review]

**Pull request merged**